# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Add first_weekday and first_workday, most come 
# DP: from OpenOffice through CLDR
# DP:    This patch applies the following conventions:
# DP:       1=Sunday...7=Saturday
# DP:       The week keyword is removed from all locales.
# DP:    Normally GTK apps should work with these settings, as well as
# DP:    other apps.
# DP: Related bugs: #343885 347323 #379100
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted

---
 localedata/locales/aa_DJ          |    2 ++
 localedata/locales/aa_ER          |    2 ++
 localedata/locales/aa_ER@saaho    |    2 ++
 localedata/locales/aa_ET          |    2 ++
 localedata/locales/af_ZA          |    2 ++
 localedata/locales/am_ET          |    2 ++
 localedata/locales/ar_AE          |    2 ++
 localedata/locales/ar_BH          |    2 ++
 localedata/locales/ar_DZ          |    2 ++
 localedata/locales/ar_EG          |    2 ++
 localedata/locales/ar_IN          |    2 ++
 localedata/locales/ar_IQ          |    2 ++
 localedata/locales/ar_JO          |    2 ++
 localedata/locales/ar_KW          |    2 ++
 localedata/locales/ar_LB          |    2 ++
 localedata/locales/ar_LY          |    2 ++
 localedata/locales/ar_MA          |    2 ++
 localedata/locales/ar_OM          |    2 ++
 localedata/locales/ar_QA          |    2 ++
 localedata/locales/ar_SA          |    2 ++
 localedata/locales/ar_SD          |    2 ++
 localedata/locales/ar_SY          |    2 ++
 localedata/locales/ar_TN          |    2 ++
 localedata/locales/ar_YE          |    2 ++
 localedata/locales/az_AZ          |    2 ++
 localedata/locales/be_BY          |    3 +++
 localedata/locales/be_BY@latin    |    3 +++
 localedata/locales/bn_IN          |    2 ++
 localedata/locales/br_FR          |    2 ++
 localedata/locales/bs_BA          |    2 ++
 localedata/locales/byn_ER         |    2 ++
 localedata/locales/ca_ES          |    2 ++
 localedata/locales/de_BE          |    2 ++
 localedata/locales/de_LU          |    2 ++
 localedata/locales/dz_BT          |    3 ++-
 localedata/locales/el_GR          |    2 ++
 localedata/locales/en_AU          |    2 ++
 localedata/locales/en_CA          |    2 ++
 localedata/locales/en_DK          |    3 +++
 localedata/locales/en_HK          |    2 ++
 localedata/locales/en_IE          |    2 ++
 localedata/locales/en_IN          |    2 ++
 localedata/locales/en_NZ          |    2 ++
 localedata/locales/en_PH          |    2 ++
 localedata/locales/en_SG          |    2 ++
 localedata/locales/en_ZA          |    2 ++
 localedata/locales/es_AR          |    2 ++
 localedata/locales/es_BO          |    2 ++
 localedata/locales/es_CL          |    2 ++
 localedata/locales/es_DO          |    2 ++
 localedata/locales/es_EC          |    2 ++
 localedata/locales/es_GT          |    2 ++
 localedata/locales/es_HN          |    2 ++
 localedata/locales/es_MX          |    2 ++
 localedata/locales/es_NI          |    2 ++
 localedata/locales/es_PA          |    2 ++
 localedata/locales/es_PE          |    2 ++
 localedata/locales/es_PR          |    2 ++
 localedata/locales/es_PY          |    2 ++
 localedata/locales/es_SV          |    2 ++
 localedata/locales/es_US          |    2 ++
 localedata/locales/es_UY          |    2 ++
 localedata/locales/es_VE          |    2 ++
 localedata/locales/eu_ES          |    1 +
 localedata/locales/ga_IE          |    2 ++
 localedata/locales/gd_GB          |    2 ++
 localedata/locales/gez_ER         |    2 ++
 localedata/locales/gez_ET         |    2 ++
 localedata/locales/gl_ES          |    2 ++
 localedata/locales/gu_IN          |    2 ++
 localedata/locales/gv_GB          |    2 ++
 localedata/locales/he_IL          |    2 ++
 localedata/locales/hi_IN          |    2 ++
 localedata/locales/hr_HR          |    2 ++
 localedata/locales/hsb_DE         |    2 ++
 localedata/locales/hy_AM          |    2 ++
 localedata/locales/id_ID          |    2 ++
 localedata/locales/is_IS          |    2 ++
 localedata/locales/it_CH          |    2 ++
 localedata/locales/it_IT          |    2 ++
 localedata/locales/ja_JP          |    2 ++
 localedata/locales/kk_KZ          |    2 ++
 localedata/locales/km_KH          |    2 ++
 localedata/locales/kn_IN          |    2 ++
 localedata/locales/ko_KR          |    2 ++
 localedata/locales/ku_TR          |    2 ++
 localedata/locales/kw_GB          |    2 ++
 localedata/locales/lo_LA          |    2 ++
 localedata/locales/lt_LT          |    2 ++
 localedata/locales/lv_LV          |    2 ++
 localedata/locales/ml_IN          |    2 ++
 localedata/locales/mn_MN          |    2 ++
 localedata/locales/mr_IN          |    2 ++
 localedata/locales/ms_MY          |    2 ++
 localedata/locales/ne_NP          |    2 ++
 localedata/locales/nl_BE          |    2 ++
 localedata/locales/nl_NL          |    2 ++
 localedata/locales/nr_ZA          |    2 ++
 localedata/locales/nso_ZA         |    2 ++
 localedata/locales/oc_FR          |    2 ++
 localedata/locales/om_ET          |    2 ++
 localedata/locales/pa_IN          |    2 ++
 localedata/locales/pt_BR          |    1 +
 localedata/locales/pt_PT          |    2 ++
 localedata/locales/rw_RW          |    2 ++
 localedata/locales/sa_IN          |    2 ++
 localedata/locales/se_NO          |    3 +++
 localedata/locales/sid_ET         |    2 ++
 localedata/locales/sl_SI          |    2 ++
 localedata/locales/so_DJ          |    2 ++
 localedata/locales/so_ET          |    2 ++
 localedata/locales/so_KE          |    2 ++
 localedata/locales/so_SO          |    2 ++
 localedata/locales/sq_AL          |    2 ++
 localedata/locales/ss_ZA          |    2 ++
 localedata/locales/st_ZA          |    2 ++
 localedata/locales/sv_FI          |    2 ++
 localedata/locales/ta_IN          |    2 ++
 localedata/locales/te_IN          |    2 ++
 localedata/locales/th_TH          |    2 ++
 localedata/locales/ti_ER          |    2 ++
 localedata/locales/ti_ET          |    2 ++
 localedata/locales/tig_ER         |    2 ++
 localedata/locales/tn_ZA          |    2 ++
 localedata/locales/tr_TR          |    2 ++
 localedata/locales/ts_ZA          |    2 ++
 localedata/locales/tt_RU          |    2 ++
 localedata/locales/uz_UZ          |    2 ++
 localedata/locales/uz_UZ@cyrillic |    2 ++
 localedata/locales/ve_ZA          |    2 ++
 localedata/locales/vi_VN          |    2 ++
 localedata/locales/wa_BE          |    3 ++-
 localedata/locales/wal_ET         |    2 ++
 localedata/locales/xh_ZA          |    2 ++
 localedata/locales/yi_US          |    2 ++
 localedata/locales/zh_CN          |    2 ++
 localedata/locales/zh_HK          |    2 ++
 localedata/locales/zh_SG          |    2 ++
 localedata/locales/zh_TW          |    3 +++
 localedata/locales/zu_ZA          |    2 ++
 143 files changed, 289 insertions(+), 2 deletions(-)

Index: glibc-2.14.1/localedata/locales/aa_DJ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/aa_DJ
+++ glibc-2.14.1/localedata/locales/aa_DJ
@@ -273,5 +273,7 @@ d_t_fmt    "<U0025><U0061><U0020><U0025>
 date_fmt    "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/aa_ER
===================================================================
--- glibc-2.14.1.orig/localedata/locales/aa_ER
+++ glibc-2.14.1/localedata/locales/aa_ER
@@ -223,5 +223,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/aa_ER@saaho
===================================================================
--- glibc-2.14.1.orig/localedata/locales/aa_ER@saaho
+++ glibc-2.14.1/localedata/locales/aa_ER@saaho
@@ -207,5 +207,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/aa_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/aa_ET
+++ glibc-2.14.1/localedata/locales/aa_ET
@@ -226,5 +226,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/af_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/af_ZA
+++ glibc-2.14.1/localedata/locales/af_ZA
@@ -164,6 +164,8 @@ t_fmt_ampm  ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/am_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/am_ET
+++ glibc-2.14.1/localedata/locales/am_ET
@@ -1564,5 +1564,7 @@ date_fmt    "<U0025><U0041><U1363><U0020
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/ar_AE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_AE
+++ glibc-2.14.1/localedata/locales/ar_AE
@@ -141,6 +141,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_BH
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_BH
+++ glibc-2.14.1/localedata/locales/ar_BH
@@ -145,6 +145,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_DZ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_DZ
+++ glibc-2.14.1/localedata/locales/ar_DZ
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_EG
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_EG
+++ glibc-2.14.1/localedata/locales/ar_EG
@@ -148,6 +148,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_IN
+++ glibc-2.14.1/localedata/locales/ar_IN
@@ -127,6 +127,8 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: copied from other *_IN locales
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_IQ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_IQ
+++ glibc-2.14.1/localedata/locales/ar_IQ
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_JO
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_JO
+++ glibc-2.14.1/localedata/locales/ar_JO
@@ -162,6 +162,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_KW
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_KW
+++ glibc-2.14.1/localedata/locales/ar_KW
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_LB
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_LB
+++ glibc-2.14.1/localedata/locales/ar_LB
@@ -161,6 +161,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_LY
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_LY
+++ glibc-2.14.1/localedata/locales/ar_LY
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_MA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_MA
+++ glibc-2.14.1/localedata/locales/ar_MA
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_OM
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_OM
+++ glibc-2.14.1/localedata/locales/ar_OM
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_QA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_QA
+++ glibc-2.14.1/localedata/locales/ar_QA
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_SA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_SA
+++ glibc-2.14.1/localedata/locales/ar_SA
@@ -347,6 +347,8 @@ era_d_fmt	""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_SD
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_SD
+++ glibc-2.14.1/localedata/locales/ar_SD
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_SY
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_SY
+++ glibc-2.14.1/localedata/locales/ar_SY
@@ -161,6 +161,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 5
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_TN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_TN
+++ glibc-2.14.1/localedata/locales/ar_TN
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ar_YE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ar_YE
+++ glibc-2.14.1/localedata/locales/ar_YE
@@ -144,6 +144,8 @@ t_fmt_ampm  "<U0025><U005A><U0020><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/az_AZ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/az_AZ
+++ glibc-2.14.1/localedata/locales/az_AZ
@@ -274,6 +274,8 @@ d_fmt     "<U0025><U0064><U002E><U0025><
 t_fmt     "<U0025><U0054>"
 am_pm     "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/be_BY
===================================================================
--- glibc-2.14.1.orig/localedata/locales/be_BY
+++ glibc-2.14.1/localedata/locales/be_BY
@@ -145,6 +145,9 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  Copied from be_BY@alternative
+first_weekday 2
+first_workday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/be_BY@latin
===================================================================
--- glibc-2.14.1.orig/localedata/locales/be_BY@latin
+++ glibc-2.14.1/localedata/locales/be_BY@latin
@@ -102,6 +102,9 @@ t_fmt_ampm ""
 date_fmt        "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  Copied from be_BY@alternative
+first_weekday 2
+first_workday 2
 END LC_TIME
 
 LC_COLLATE
Index: glibc-2.14.1/localedata/locales/bn_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/bn_IN
+++ glibc-2.14.1/localedata/locales/bn_IN
@@ -145,6 +145,8 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/br_FR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/br_FR
+++ glibc-2.14.1/localedata/locales/br_FR
@@ -141,6 +141,8 @@ d_fmt   "<U0025><U0064><U002E><U0025><U0
 t_fmt   "<U0025><U0054>"
 am_pm   "<U0020>";"<U0020>"
 t_fmt_ampm "<U0025><U0049><U0065><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/bs_BA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/bs_BA
+++ glibc-2.14.1/localedata/locales/bs_BA
@@ -208,6 +208,8 @@ d_fmt   "<U0025><U0064><U002E><U0025><U0
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/byn_ER
===================================================================
--- glibc-2.14.1.orig/localedata/locales/byn_ER
+++ glibc-2.14.1/localedata/locales/byn_ER
@@ -218,5 +218,7 @@ date_fmt    "<U0025><U0041><U1361><U0020
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12A3><U12F5>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/ca_ES
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ca_ES
+++ glibc-2.14.1/localedata/locales/ca_ES
@@ -138,6 +138,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/de_BE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/de_BE
+++ glibc-2.14.1/localedata/locales/de_BE
@@ -119,6 +119,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/de_LU
===================================================================
--- glibc-2.14.1.orig/localedata/locales/de_LU
+++ glibc-2.14.1/localedata/locales/de_LU
@@ -119,6 +119,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/dz_BT
===================================================================
--- glibc-2.14.1.orig/localedata/locales/dz_BT
+++ glibc-2.14.1/localedata/locales/dz_BT
@@ -606,7 +606,8 @@ t_fmt_ampm        "<U0F46><U0F74><U0F0B>
 <U0025><U0049><U0F40><U0F66><U0F62><U0F0B><U0F58><U0025><U004D>/
 <U0F40><U0F66><U0F62><U0F0B><U0F46><U0025><U0053><U0020><U0025><U0070>"
 
-
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/el_GR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/el_GR
+++ glibc-2.14.1/localedata/locales/el_GR
@@ -139,6 +139,8 @@ t_fmt_ampm "<U0025><U0049><U003A><U0025>
 %
 % Appropriate date representation (date(1))   "%a %d %b %Y %r %Z"
 date_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0072><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/en_AU
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_AU
+++ glibc-2.14.1/localedata/locales/en_AU
@@ -117,6 +117,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/en_CA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_CA
+++ glibc-2.14.1/localedata/locales/en_CA
@@ -132,6 +132,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/en_HK
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_HK
+++ glibc-2.14.1/localedata/locales/en_HK
@@ -133,6 +133,8 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0070><U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/en_IE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_IE
+++ glibc-2.14.1/localedata/locales/en_IE
@@ -115,6 +115,8 @@ t_fmt_ampm  ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/en_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_IN
+++ glibc-2.14.1/localedata/locales/en_IN
@@ -132,6 +132,8 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/en_NZ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_NZ
+++ glibc-2.14.1/localedata/locales/en_NZ
@@ -117,6 +117,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/en_PH
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_PH
+++ glibc-2.14.1/localedata/locales/en_PH
@@ -133,6 +133,8 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/en_SG
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_SG
+++ glibc-2.14.1/localedata/locales/en_SG
@@ -132,6 +132,8 @@ t_fmt       "<U0025><U0054>"
 % %I:%M:%S %p
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/en_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/en_ZA
+++ glibc-2.14.1/localedata/locales/en_ZA
@@ -205,6 +205,8 @@ t_fmt_ampm  ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/es_AR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_AR
+++ glibc-2.14.1/localedata/locales/es_AR
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_BO
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_BO
+++ glibc-2.14.1/localedata/locales/es_BO
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_CL
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_CL
+++ glibc-2.14.1/localedata/locales/es_CL
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_DO
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_DO
+++ glibc-2.14.1/localedata/locales/es_DO
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_EC
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_EC
+++ glibc-2.14.1/localedata/locales/es_EC
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_GT
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_GT
+++ glibc-2.14.1/localedata/locales/es_GT
@@ -121,6 +121,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_HN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_HN
+++ glibc-2.14.1/localedata/locales/es_HN
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_MX
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_MX
+++ glibc-2.14.1/localedata/locales/es_MX
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_NI
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_NI
+++ glibc-2.14.1/localedata/locales/es_NI
@@ -114,6 +114,8 @@ am_pm	"<U0041><U004D>";"<U0050><U004D>"
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_PA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_PA
+++ glibc-2.14.1/localedata/locales/es_PA
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_PE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_PE
+++ glibc-2.14.1/localedata/locales/es_PE
@@ -123,6 +123,8 @@ am_pm	"<U0041><U004D>";"<U0050><U004D>"
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_PR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_PR
+++ glibc-2.14.1/localedata/locales/es_PR
@@ -113,6 +113,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_PY
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_PY
+++ glibc-2.14.1/localedata/locales/es_PY
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_SV
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_SV
+++ glibc-2.14.1/localedata/locales/es_SV
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_US
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_US
+++ glibc-2.14.1/localedata/locales/es_US
@@ -174,6 +174,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME
+first_weekday 1
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_UY
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_UY
+++ glibc-2.14.1/localedata/locales/es_UY
@@ -122,6 +122,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/es_VE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/es_VE
+++ glibc-2.14.1/localedata/locales/es_VE
@@ -123,6 +123,8 @@ t_fmt_ampm "<U0025><U0049><U003A><U0025>
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/eu_ES
===================================================================
--- glibc-2.14.1.orig/localedata/locales/eu_ES
+++ glibc-2.14.1/localedata/locales/eu_ES
@@ -131,6 +131,7 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/ga_IE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ga_IE
+++ glibc-2.14.1/localedata/locales/ga_IE
@@ -117,6 +117,8 @@ t_fmt_ampm  ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/gd_GB
===================================================================
--- glibc-2.14.1.orig/localedata/locales/gd_GB
+++ glibc-2.14.1/localedata/locales/gd_GB
@@ -122,6 +122,8 @@ d_fmt       "<U0025><U0064><U002F><U0025
 t_fmt       "<U0025><U0054>"
 am_pm       "";""
 t_fmt_ampm  ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/gez_ER
===================================================================
--- glibc-2.14.1.orig/localedata/locales/gez_ER
+++ glibc-2.14.1/localedata/locales/gez_ER
@@ -215,5 +215,7 @@ date_fmt    "<U0025><U0041><U1365><U0025
 <U1361><U0025><U0072><U1361><U0025><U005A><U1361><U0025><U0059>/
 <U1361><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/gez_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/gez_ET
+++ glibc-2.14.1/localedata/locales/gez_ET
@@ -216,5 +216,7 @@ date_fmt    "<U0025><U0041><U1365><U0025
 <U1361><U0025><U0072><U1361><U0025><U005A><U1361><U0025><U0059>/
 <U1361><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/gl_ES
===================================================================
--- glibc-2.14.1.orig/localedata/locales/gl_ES
+++ glibc-2.14.1/localedata/locales/gl_ES
@@ -121,6 +121,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/gu_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/gu_IN
+++ glibc-2.14.1/localedata/locales/gu_IN
@@ -178,6 +178,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/gv_GB
===================================================================
--- glibc-2.14.1.orig/localedata/locales/gv_GB
+++ glibc-2.14.1/localedata/locales/gv_GB
@@ -131,6 +131,8 @@ t_fmt_ampm  ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/he_IL
===================================================================
--- glibc-2.14.1.orig/localedata/locales/he_IL
+++ glibc-2.14.1/localedata/locales/he_IL
@@ -123,6 +123,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/hi_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/hi_IN
+++ glibc-2.14.1/localedata/locales/hi_IN
@@ -173,6 +173,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/hr_HR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/hr_HR
+++ glibc-2.14.1/localedata/locales/hr_HR
@@ -2212,6 +2212,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/hy_AM
===================================================================
--- glibc-2.14.1.orig/localedata/locales/hy_AM
+++ glibc-2.14.1/localedata/locales/hy_AM
@@ -147,6 +147,8 @@ d_fmt       "<U0025><U006D><U002F><U0025
 t_fmt       "<U0025><U0072>"
 am_pm       "";""
 t_fmt_ampm  ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/id_ID
===================================================================
--- glibc-2.14.1.orig/localedata/locales/id_ID
+++ glibc-2.14.1/localedata/locales/id_ID
@@ -120,6 +120,8 @@ t_fmt_ampm  ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/is_IS
===================================================================
--- glibc-2.14.1.orig/localedata/locales/is_IS
+++ glibc-2.14.1/localedata/locales/is_IS
@@ -2223,6 +2223,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in Java
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/it_CH
===================================================================
--- glibc-2.14.1.orig/localedata/locales/it_CH
+++ glibc-2.14.1/localedata/locales/it_CH
@@ -100,6 +100,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0065><U0020><U0025>/
 <U0062><U0020><U0025><U0059><U002C><U0020><U0025><U0048><U002E>/
 <U0025><U004D><U002E><U0025><U0053><U002C><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/it_IT
===================================================================
--- glibc-2.14.1.orig/localedata/locales/it_IT
+++ glibc-2.14.1/localedata/locales/it_IT
@@ -124,6 +124,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0065><U0020><U0025>/
 <U0062><U0020><U0025><U0059><U002C><U0020><U0025><U0048><U002E>/
 <U0025><U004D><U002E><U0025><U0053><U002C><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/ja_JP
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ja_JP
+++ glibc-2.14.1/localedata/locales/ja_JP
@@ -15006,6 +15006,8 @@ alt_digits	"<U3007>";"<U4E00>";"<U4E8C>"
 		"<U4E5D><U5341><U56DB>";"<U4E5D><U5341><U4E94>";/
 		"<U4E5D><U5341><U516D>";"<U4E5D><U5341><U4E03>";/
 		"<U4E5D><U5341><U516B>";"<U4E5D><U5341><U4E5D>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/kk_KZ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/kk_KZ
+++ glibc-2.14.1/localedata/locales/kk_KZ
@@ -232,6 +232,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/km_KH
===================================================================
--- glibc-2.14.1.orig/localedata/locales/km_KH
+++ glibc-2.14.1/localedata/locales/km_KH
@@ -1841,6 +1841,8 @@ am_pm    "<U1796><U17D2><U179A><U17B9><U
 % ICT-7ICT-7
 %timezone	"<U0049><U0043><U0054><U002D><U0037><U0049><U0043><U0054><U002d><U0037>"
 
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/kn_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/kn_IN
+++ glibc-2.14.1/localedata/locales/kn_IN
@@ -181,6 +181,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ko_KR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ko_KR
+++ glibc-2.14.1/localedata/locales/ko_KR
@@ -6189,6 +6189,8 @@ date_fmt  /
 <U0020><U0028><U0025><U0061><U0029>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>/
 <U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 1
 
 END LC_TIME
 
Index: glibc-2.14.1/localedata/locales/ku_TR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ku_TR
+++ glibc-2.14.1/localedata/locales/ku_TR
@@ -154,6 +154,8 @@ am_pm       "";""
 t_fmt_ampm  ""
 % format: %A %d %B %Y
 date_fmt  "<U0025><U0041><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/kw_GB
===================================================================
--- glibc-2.14.1.orig/localedata/locales/kw_GB
+++ glibc-2.14.1/localedata/locales/kw_GB
@@ -123,6 +123,8 @@ t_fmt_ampm  ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/lo_LA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/lo_LA
+++ glibc-2.14.1/localedata/locales/lo_LA
@@ -718,6 +718,8 @@ date_fmt       "<U0025><U0061><U0020><U0
 <U003A><U0025><U0053><U0020><U0025><U005A>"
 % ICT-7ICT-7
 timezone	"<U0049><U0043><U0054><U002D><U0037><U0049><U0043><U0054><U002d><U0037>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/lt_LT
===================================================================
--- glibc-2.14.1.orig/localedata/locales/lt_LT
+++ glibc-2.14.1/localedata/locales/lt_LT
@@ -2191,6 +2191,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/lv_LV
===================================================================
--- glibc-2.14.1.orig/localedata/locales/lv_LV
+++ glibc-2.14.1/localedata/locales/lv_LV
@@ -2183,6 +2183,8 @@ t_fmt_ampm  ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/ml_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ml_IN
+++ glibc-2.14.1/localedata/locales/ml_IN
@@ -163,6 +163,8 @@ t_fmt     "<U0025><U0049><U003A><U0025><
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm     "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME
+first_weekday 1
 END LC_TIME
 %
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/mn_MN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/mn_MN
+++ glibc-2.14.1/localedata/locales/mn_MN
@@ -220,6 +220,8 @@ t_fmt_ampm ""
 % "%Z %Y ony %B %e, %a %H:%M:%S"
 date_fmt  "<U0025><U005A><U0020><U0025><U0059><U0020><U043E><U043D><U044B><U0020><U0025><U0042><U0020>/
 <U0025><U0065><U002C><U0020><U0025><U0061><U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/mr_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/mr_IN
+++ glibc-2.14.1/localedata/locales/mr_IN
@@ -170,6 +170,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ms_MY
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ms_MY
+++ glibc-2.14.1/localedata/locales/ms_MY
@@ -139,6 +139,8 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ne_NP
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ne_NP
+++ glibc-2.14.1/localedata/locales/ne_NP
@@ -153,6 +153,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/nl_BE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/nl_BE
+++ glibc-2.14.1/localedata/locales/nl_BE
@@ -104,6 +104,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/nr_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/nr_ZA
+++ glibc-2.14.1/localedata/locales/nr_ZA
@@ -162,6 +162,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/nso_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/nso_ZA
+++ glibc-2.14.1/localedata/locales/nso_ZA
@@ -157,6 +157,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
  
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/oc_FR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/oc_FR
+++ glibc-2.14.1/localedata/locales/oc_FR
@@ -123,6 +123,8 @@ d_fmt   "<U0025><U0064><U002E><U0025><U0
 t_fmt   "<U0025><U0054>"
 am_pm   "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_NAME
Index: glibc-2.14.1/localedata/locales/om_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/om_ET
+++ glibc-2.14.1/localedata/locales/om_ET
@@ -233,5 +233,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/pa_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/pa_IN
+++ glibc-2.14.1/localedata/locales/pa_IN
@@ -170,6 +170,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/pt_BR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/pt_BR
+++ glibc-2.14.1/localedata/locales/pt_BR
@@ -120,6 +120,7 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+first_weekday 1
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/pt_PT
===================================================================
--- glibc-2.14.1.orig/localedata/locales/pt_PT
+++ glibc-2.14.1/localedata/locales/pt_PT
@@ -124,6 +124,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/rw_RW
===================================================================
--- glibc-2.14.1.orig/localedata/locales/rw_RW
+++ glibc-2.14.1/localedata/locales/rw_RW
@@ -109,6 +109,8 @@ t_fmt_ampm ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/sa_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/sa_IN
+++ glibc-2.14.1/localedata/locales/sa_IN
@@ -182,6 +182,8 @@ t_fmt_ampm  "<U0025><U0049><U003A><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: copied from other *_IN locales
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/se_NO
===================================================================
--- glibc-2.14.1.orig/localedata/locales/se_NO
+++ glibc-2.14.1/localedata/locales/se_NO
@@ -276,6 +276,9 @@ date_fmt       "<U0025><U0041><U002C><U0
 <U0020><U0025><U005A>"
 
 % %a %b-%e %H:%M:%S %Z %Y sn jan-31 21:15:11 CET 2001
+%  FIXME: found in CLDR
+first_weekday 2
+first_workday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/sid_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/sid_ET
+++ glibc-2.14.1/localedata/locales/sid_ET
@@ -222,5 +222,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 %
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/so_DJ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/so_DJ
+++ glibc-2.14.1/localedata/locales/so_DJ
@@ -222,5 +222,7 @@ d_t_fmt    "<U0025><U0061><U0020><U0025>
 date_fmt    "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/so_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/so_ET
+++ glibc-2.14.1/localedata/locales/so_ET
@@ -219,5 +219,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/so_KE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/so_KE
+++ glibc-2.14.1/localedata/locales/so_KE
@@ -218,5 +218,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/so_SO
===================================================================
--- glibc-2.14.1.orig/localedata/locales/so_SO
+++ glibc-2.14.1/localedata/locales/so_SO
@@ -278,5 +278,7 @@ d_t_fmt    "<U0025><U0041><U002C><U0020>
 date_fmt    "<U0025><U0041><U002C><U0020><U0025><U0042><U0020><U0025><U0065><U002C>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/sq_AL
===================================================================
--- glibc-2.14.1.orig/localedata/locales/sq_AL
+++ glibc-2.14.1/localedata/locales/sq_AL
@@ -290,6 +290,8 @@ t_fmt_ampm  "<U0025><U0049><U002E><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/ss_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ss_ZA
+++ glibc-2.14.1/localedata/locales/ss_ZA
@@ -167,6 +167,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/st_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/st_ZA
+++ glibc-2.14.1/localedata/locales/st_ZA
@@ -164,6 +164,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/ta_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ta_IN
+++ glibc-2.14.1/localedata/locales/ta_IN
@@ -154,6 +154,8 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/te_IN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/te_IN
+++ glibc-2.14.1/localedata/locales/te_IN
@@ -156,6 +156,8 @@ t_fmt       "<U0025><U0070><U0025><U0049
 % %p%I.%M.%S %Z
 t_fmt_ampm  "<U0025><U0070><U0025><U0049><U002E><U0025><U004D><U002E><U0025><U0053><U0020><U0025><U005A>"
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/th_TH
===================================================================
--- glibc-2.14.1.orig/localedata/locales/th_TH
+++ glibc-2.14.1/localedata/locales/th_TH
@@ -913,6 +913,8 @@ date_fmt       "<U0025><U0061><U0020><U0
 <U003A><U0025><U0053><U0020><U0025><U005A>"
 % ICT-7ICT-7
 timezone	"<U0049><U0043><U0054><U002D><U0037><U0049><U0043><U0054><U002d><U0037>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/ti_ER
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ti_ER
+++ glibc-2.14.1/localedata/locales/ti_ER
@@ -270,5 +270,7 @@ date_fmt    "<U0025><U0041><U1361><U0020
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/ti_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ti_ET
+++ glibc-2.14.1/localedata/locales/ti_ET
@@ -1072,5 +1072,7 @@ date_fmt    "<U0025><U0041><U1363><U0020
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/tig_ER
===================================================================
--- glibc-2.14.1.orig/localedata/locales/tig_ER
+++ glibc-2.14.1/localedata/locales/tig_ER
@@ -218,5 +218,7 @@ date_fmt    "<U0025><U0041><U1361><U0020
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U12D3><U002F><U121D>"
 %
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/tn_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/tn_ZA
+++ glibc-2.14.1/localedata/locales/tn_ZA
@@ -169,6 +169,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/tr_TR
===================================================================
--- glibc-2.14.1.orig/localedata/locales/tr_TR
+++ glibc-2.14.1/localedata/locales/tr_TR
@@ -3540,6 +3540,8 @@ t_fmt_ampm "<U0025><U0049><U003A><U0025>
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/ts_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ts_ZA
+++ glibc-2.14.1/localedata/locales/ts_ZA
@@ -164,6 +164,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/tt_RU
===================================================================
--- glibc-2.14.1.orig/localedata/locales/tt_RU
+++ glibc-2.14.1/localedata/locales/tt_RU
@@ -297,6 +297,8 @@ d_fmt     "<U0025><U0064><U002E><U0025><
 t_fmt     "<U0025><U0054>"
 am_pm     "";""
 t_fmt_ampm ""
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
Index: glibc-2.14.1/localedata/locales/uz_UZ
===================================================================
--- glibc-2.14.1.orig/localedata/locales/uz_UZ
+++ glibc-2.14.1/localedata/locales/uz_UZ
@@ -260,6 +260,8 @@ t_fmt       "<U0025><U0054>"
 am_pm       "";""
 t_fmt_ampm  ""
 date_fmt    "<U0025><U0064><U0020><U0025><U0042><U002C><U0020><U0025><U0059><U0020><U0079><U0069><U006C><U002C><U0020><U0025><U0041>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/uz_UZ@cyrillic
===================================================================
--- glibc-2.14.1.orig/localedata/locales/uz_UZ@cyrillic
+++ glibc-2.14.1/localedata/locales/uz_UZ@cyrillic
@@ -263,6 +263,8 @@ t_fmt       "<U0025><U0054>"
 am_pm       "";""
 t_fmt_ampm  ""
 date_fmt    "<U0025><U0064><U0020><U0025><U0042><U002C><U0020><U0025><U0059><U0020><U0439><U0438><U043B><U002C><U0020><U0025><U0041>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/ve_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/ve_ZA
+++ glibc-2.14.1/localedata/locales/ve_ZA
@@ -166,6 +166,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/vi_VN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/vi_VN
+++ glibc-2.14.1/localedata/locales/vi_VN
@@ -294,6 +294,8 @@ t_fmt    "<U0025><U0054>"
 am_pm    "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U0020><U0025><U0070>"
 date_fmt  "<U0025><U0041><U002C><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U006E><U0103><U006D><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 % yes: Va^ng / Da.
Index: glibc-2.14.1/localedata/locales/wa_BE
===================================================================
--- glibc-2.14.1.orig/localedata/locales/wa_BE
+++ glibc-2.14.1/localedata/locales/wa_BE
@@ -163,7 +163,8 @@ t_fmt   "<U0025><U0048><U003A><U0025><U0
 am_pm   "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070>"
 % prum djo del samwinne (1=londi)
-first_weekday 1
+% FIXME  No, 2=lundi
+first_weekday 2
 END LC_TIME
 
 LC_NAME
Index: glibc-2.14.1/localedata/locales/wal_ET
===================================================================
--- glibc-2.14.1.orig/localedata/locales/wal_ET
+++ glibc-2.14.1/localedata/locales/wal_ET
@@ -233,5 +233,7 @@ date_fmt    "<U0025><U0041><U1363><U0020
 <U0020><U130B><U120B><U1233>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U130D><U002F><U120B>"
+%  FIXME: found in CLDR
+first_weekday 7
 END LC_TIME
 %
Index: glibc-2.14.1/localedata/locales/xh_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/xh_ZA
+++ glibc-2.14.1/localedata/locales/xh_ZA
@@ -164,6 +164,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % time-zone name,
 % year with century as a decimal number,e.g. 2001.
 %
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/yi_US
===================================================================
--- glibc-2.14.1.orig/localedata/locales/yi_US
+++ glibc-2.14.1/localedata/locales/yi_US
@@ -186,6 +186,8 @@ t_fmt   "<U0025><U0048><U003A><U0025><U0
 am_pm   "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0050>"
 cal_direction	3
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/zh_CN
===================================================================
--- glibc-2.14.1.orig/localedata/locales/zh_CN
+++ glibc-2.14.1/localedata/locales/zh_CN
@@ -135,6 +135,8 @@ t_fmt_ampm "<U0025><U0070><U0020><U0025>
 date_fmt       "<U0025><U0059><U5E74><U0020><U0025><U006D><U6708><U0020>/
 <U0025><U0064><U65E5><U0020><U0025><U0041><U0020><U0025><U0048><U003A>/
 <U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/zh_HK
===================================================================
--- glibc-2.14.1.orig/localedata/locales/zh_HK
+++ glibc-2.14.1/localedata/locales/zh_HK
@@ -141,6 +141,8 @@ t_fmt_ampm  "<U0025><U0070><U0020><U0025
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/zh_SG
===================================================================
--- glibc-2.14.1.orig/localedata/locales/zh_SG
+++ glibc-2.14.1/localedata/locales/zh_SG
@@ -130,6 +130,8 @@ t_fmt       "<U0025><U0048><U65F6><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  ""
 %
+%  FIXME: found in CLDR
+first_weekday 2
 END LC_TIME
 
 
Index: glibc-2.14.1/localedata/locales/zh_TW
===================================================================
--- glibc-2.14.1.orig/localedata/locales/zh_TW
+++ glibc-2.14.1/localedata/locales/zh_TW
@@ -128,6 +128,9 @@ date_fmt       "<U0025><U0061><U0020><U0
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
 
+%  FIXME: found in CLDR
+first_weekday 1
+
 END LC_TIME
 
 LC_MESSAGES
Index: glibc-2.14.1/localedata/locales/zu_ZA
===================================================================
--- glibc-2.14.1.orig/localedata/locales/zu_ZA
+++ glibc-2.14.1/localedata/locales/zu_ZA
@@ -170,6 +170,8 @@ date_fmt "<U0025><U0061><U0020><U0025><U
 % %Z - time-zone name,
 % %Y - year with century as a decimal number,e.g. 2001.
 
+%  FIXME: found in CLDR
+first_weekday 1
 END LC_TIME
 
 LC_MESSAGES
